### DISCUSSION TITLE
Unknown hostname is served with self signed certificate

### DISCUSSION THREAD

**Discussion by jwgn** [tobychui/zoraxy#345] [Q&A]
When Zoraxy is hit with an unknown hostname, it should utilize the `Default Site` setting, in my case `Internal Static Web Server`.

But it throws an error instead. `The certificate is not trusted because it is self-signed.` Signed by `Zoraxy Self-host`.

How to fix this?

**Comment by tobychui** [comment]
Upload your default certificate to the "Fallback Certificate" field in the TLS/SSL management tab.

**Comment by jwgn** [comment]
How do I generate such a fallback certificate which is valid for all unknown domains? I use the ACME tool in Zoraxy to generade certificate for all my domains.

**Comment by jwgn** [comment]
Can't I just disable unknown hostnames? It would be better if there was just a connection error instead of any redirect.

**Comment by jwgn** [comment]
Nevermind, I just had to remove the wildcard DNS record :)
